packages/components/src/components/lifecycleValidation.spec.ts

```diff
@@ -14,6 +15,9 @@ const tagNamesWithRequiredChild = TAG_NAMES.filter((tagName) => getComponentMeta
 const tagNamesWithRequiredParent = TAG_NAMES.filter((tagName) => getComponentMeta(tagName).requiredParent);
 const tagNamesWithRequiredRootNode = TAG_NAMES.filter((tagName) => getComponentMeta(tagName).requiredRootNode);
 const tagNamesWithJss = TAG_NAMES.filter((tagName) => getComponentMeta(tagName).styling === 'jss');
+const tagNamesWithSlotAndTheme = TAG_NAMES.filter(
+  (tagName) => getComponentMeta(tagName).hasSlot && getComponentMeta(tagName).props?.theme
```

Comment:
I think there is a `getComponentMeta(tagName).isThemeable` which would be easier to use.